src/thunderbots/software/CMakeLists.txt

```diff
@@ -63,6 +68,7 @@ catkin_package(
 
 # Specify additional locations of header files
 include_directories(
+	SYSTEM "${LIB_DIR}/Include"
```

Comment:
I think putting `SYSTEM` here like this causes all the includes to be considered system includes (and thus ignores warnings for them). From looking at the [include_directories documentation](https://cmake.org/cmake/help/v3.0/command/include_directories.html), looks like the way to do this is just to have two `include_directories` statements, like so:
```
include_directories(SYSTEM 
    ${catkin_INCLUDE_DIRS}
    )
include_directories(
        ${LIB_DIR}/Include
        ${PROTOBUF_INCLUDE_DIRS}
        ${CMAKE_CURRENT_SOURCE_DIR}
        ${CMAKE_CURRENT_SOURCE_DIR}/..
        )
```
(also not sure if we need the quotes around `${LIB_DIR}/Include` or not)

Reply:
I think quotes are usually optional. Just pick one and be consistent.

I agree that having two statements is more clear

Reply:
fixed